server/service/src/sync/test/test_data/vaccine_course_dose.rs

```diff
@@ -12,6 +12,7 @@ const VACCINE_COURSE_DOSE1: (&str, &str) = (
         "vaccine_course_id": "test_vaccine_course",
         "label": "test dose label",
         "min_age": 12.0,
+        "min_age": 13.0,
```

Comment:
```suggestion
        "max_age": 13.0,
```
Causing a failing test...

Reply:
Done, thanks